scc/delivery-with-flux.hbs.md

```diff
@@ -165,6 +165,8 @@ Configure Flux CD on the Build cluster to deploy your `Packages`, `PackageInstal
        --from-file=value.yaml=<path-to-run-cluster-kubeconfig>
    ```
 
+   > **Note** The KubeConfig should be self-contained and not rely on binaries, environment, or credential files from the kustomize-controller Pod. KubeConfigs with cmd-path in them likely won’t work without a custom, per-provider installation of kustomize-controller.
```

Comment:
What does cmd-path mean in this context?
Should cmd-path be in code format?
Is it referring to the method where you pass the kubeconfig file with the kubectl command?
Also, should this note be added as a 1-7-4 known issue in the release notes?
Thanks

Reply:
cmd-path is the field `command` in a Kubeconfig, see https://kubernetes.io/docs/reference/config-api/kubeconfig.v1/#ExecConfig. I can change this to something more specific like
```
KubeConfigs that specify a command to provide client credentials (e.g. `gke-gcloud-auth-plugin`) won’t work without a custom, per-provider installation of kustomize-controller.
```

I wouldn't consider this a known issue, it's just getting a bit more specific with the setup instructions for Flux.